components/alert/alert.hbs

```diff
@@ -1,32 +1,102 @@
+						<use href="{{ path '/msds-spritemap.svg' }}#cross" />
+					</svg>
+				</div>
+				<div class="msds-alert__content-container">
+					<img class="msds-alert__image" src="{{ path '/src/demo-img/time-to-market.png' }}">
+					<div class="msds-alert__text-container">
+						<div class="msds-alert__header">Donec libero turpis, lacinia nec leo sit amet</div>
+						<p class="msds-alert__text">
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit
+							amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie.
+							Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac
+							porttitor pulvinar.
+						</p>
+						<div class="msds-alert__action-container">
+							<button type="button"
+								class="msds-btn msds-btn--secondary msds-btn--sm">ok,
+								i get it</button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
 	<h2>Extra attention alert - With image</h2>
 	<div class="row">
 		<div class="col-12">
 			<div class="d-none msds-alert" data-alert-id="1234">
-				<div class="msds-alert__close-click-area js-msds-alert-close-click-handler">
+				<div class="msds-alert__close-click-area">
 					<svg class="msds-alert__close-icon">
-					  <use href="{{ path '/msds-spritemap.svg' }}#cross" />
+						<use href="{{ path '/msds-spritemap.svg' }}#cross" />
 					</svg>
 				</div>
 				<div class="msds-alert__content-container">
 					<img class="msds-alert__image" src="{{ path '/src/demo-img/time-to-market.png' }}">
 					<div class="msds-alert__text-container">
 						<div class="msds-alert__header">Donec libero turpis, lacinia nec leo sit amet</div>
 						<p class="msds-alert__text">
-							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie. Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac porttitor pulvinar.
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit
+							amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie.
+							Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac
+							porttitor pulvinar.
 						</p>
 						<div class="msds-alert__action-container">
-							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm">ok, i get it</button>
- 
+							<button type="button"
+								class="msds-btn msds-btn--secondary msds-btn--sm">ok,
+								i get it</button>
+
 							<div class="msds-alert__checkbox-container">
-								<input type="checkbox" class="msds-alert__checkbox" id="msds-alert__checkbox" type="text">
-								<label class="msds-alert__checkbox-label" for="msds-alert__checkbox">Don't show this again</label>
+								<input type="checkbox" class="msds-alert__checkbox" id="msds-alert__checkbox"
```

Comment:
Would it make sense to reuse the component here in the checkbox component for the forms and then import it here aswell?

Reply:
we don't have the checkbox component yet to my knowledge but yes I agree that would be the right way to go